### PR TITLE
[TOMEE-2671] Adding mp-custom-healthcheck Spanish Translation

### PR DESCRIPTION
**Pull request by emecas**
https://issues.apache.org/jira/browse/TOMEE-2671

### CODE REVIEW

File: examples/mp-custom-healthcheck/README_es.adoc [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                    .withData("weatherServiceMessage", status.getMessage())
+                    .up().build();
+        } catch (WeatherException e) {
+            return responseBuilder.withData("weatherServiceErrorMessage", e.getMessage()).down().build();
+        }
+    }
+}
+----
+
+En el anterior ejemplo, el health end point es: https://openweathermap.org/appid[OpenWeatherMap] (_solamente ilustrativo_) que proporciona una suscripción al plan para acceder los servicios y si el limite de llamadas a la API se excede no estará disponible hasta que se renueve las suscripción.
```
Comment (sercheo87) [inline examples/mp-custom-healthcheck/README_es.adoc]: La traducción quedaria mejor de 

> el health end point es: 

a 

> el nombre de la sonda de salud es:

Comment (emecas) [reply examples/mp-custom-healthcheck/README_es.adoc]: I have never read "health end point" as "sonda de salud", I would use more something like: "enlace de estado"

Comment (sercheo87) [reply examples/mp-custom-healthcheck/README_es.adoc]: Yes, this tradition is best.

**Comment by dblevins** [comment]
Seems like this one is good now.  Gracias @emecas y @sercheo87 !

**Pull request merged**